core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -188,6 +193,12 @@ formatName: FORMAT_NAME EQ string
 formatType: TYPE EQ typeFileformat formatTypeOptions*
     ;
 
+let: LET id dataType? ASSIGN expr SEMI | LET id dataType? DEFAULT expr SEMI | id ASSIGN expr SEMI
+    ;
```

Comment:
You can simply write:
```suggestion
let: LET id dataType? ((ASSIGN | DEFAULT) expr)? SEMI 
    ;
```
Or use labels (as described above), whatever makes the visitor implementation easier.

Reply:
Taken care